src/thor/bidirectional_astar.cc

```diff
@@ -507,8 +507,9 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
   PointLL destination_new(destination.path_edges(0).ll().lng(), destination.path_edges(0).ll().lat());
   Init(origin_new, destination_new);
 
+  // Set invariant true for bidirectional a*, change it when the algo allows varying time.
+  bool invariant = true;
```

Comment:
this doesnt seem right, because you can still make a route that doesnt ask for timedependence. i didnt think about it all the way but i was kind of expecting this to be the change:

    // we use a non varying time for all time dependent routes until we can figure out how to vary the time during
    // the path computation in the bidirectional algorithm
    bool invariant = options.has_date_time_type();

Reply:
so actually i guess we can completely delete this boolean and modify `Expand` directly... let me make another comment elsewhere in the diff

Reply:
maybe the variable would be better labeled as `time_dependent` and then over in `Expect` we could add the comment that currently bidira* cant vary the time for time dependent routes so we use `0` seconds for offset? any thoughts to make this more clear ))

Reply:
I would leave it invariant, it is more clear what it is created for in this case. Once we update the algo and time updates will be possible, we can separate behaviour. Now the behaviour is more visible.

@kevinkreiser What did you mean by `Expect` comment? I've updated the documentations to emphasize this behaviour.

Reply:
Sorry it was a very stupid typo, kids were probably yelling in the background at the time... I meant the `Expand` method haha sorry for that

Reply:
And yeah I'm fine leaving it I just thought some way to call attention to that spot would be good so future us will understand/remember it